### PR TITLE
NIAD-97 fake-spine for v&p testing

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

File: integration-tests/fake_spine/fake_spine/tests/test_vnp_spine_response.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import unittest
```
Comment (mattd-kainos) [inline integration-tests/fake_spine/fake_spine/tests/test_vnp_spine_response.py]: Do these tests run in the pipeline?

File: integration-tests/fake_spine/fake_spine/pystache_message_builder.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+"""A module that defines a Pystache-based MessageBuilder."""
```
Comment (mattd-kainos) [inline integration-tests/fake_spine/fake_spine/pystache_message_builder.py:1]: I copied this file over from integration-adaptors-common. Not idea I know. The fake-spine project is setup is currently independent of the rest of the code base. We'd need to change the Dockerfile and some of the build to bring in other modules as a dependency.

**Review by grzegorz-robak — DISMISSED** [review]

**Review by grzegorz-robak — DISMISSED** [review]

File: integration-tests/fake_spine/fake_spine/vnp_spine_message_builder.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+import os
+from pathlib import Path
+
+from fake_spine.pystache_message_builder import PystacheMessageBuilder
+
+TEMPLATES_DIR = "vnp_responses"
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```
Comment (michael-kainos) [inline integration-tests/fake_spine/fake_spine/vnp_spine_message_builder.py]: Seen this across a few files, should this be imported from another class?

Comment (mattd-kainos) [reply integration-tests/fake_spine/fake_spine/vnp_spine_message_builder.py]: Good spot it is done this way in some other modules

Comment (mattd-kainos) [reply integration-tests/fake_spine/fake_spine/vnp_spine_message_builder.py]: Done!

**Review by grzegorz-robak — DISMISSED** [review]

**Review by michael-kainos — APPROVED** [review]

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Pull request merged**